src/herder/TxQueueLimiter.cpp

```diff
@@ -103,11 +103,14 @@ TxQueueLimiter::canAddTx(TransactionFrameBasePtr& newTx,
     auto txBaseFee = newTx->getBaseFeeBid();
     if (txBaseFee <= getMinBaseFee())
     {
-        return false;
+        auto minFee = newTx->getFee(Config::CURRENT_LEDGER_PROTOCOL_VERSION,
+                                    txBaseFee, false) +
```

Comment:
This doesn't seem right, shouldn't it say `getMinBaseFee()` here?

Reply:
removed